client/src/components/cullingexclusionzones/CullingExclusionZones.tsx

```diff
@@ -0,0 +1,47 @@
+import { UnculledZone } from "../../api/liberationApi";
+import { selectUnculledZones } from "../../api/unculledZonesSlice";
+import { useAppSelector } from "../../app/hooks";
+import { LayerGroup, LayersControl, Circle } from "react-leaflet";
+
+interface CullingExclusionCirclesProps {
+  zones: UnculledZone[];
+}
+
+const CullingExclusionCircles = (props: CullingExclusionCirclesProps) => {
+  return (
+    <>
+      <LayerGroup>
+        {props.zones.map((zone, idx) => {
+          return (
+            <Circle
+              key={idx}
+              center={zone.position}
+              radius={zone.radius}
+              color="#b4ff8c"
+              fill={false}
+              interactive={false}
+            />
+          );
+        })}
+      </LayerGroup>
+    </>
+  );
+};
+
+export default function CullingExclusionZones() {
+  const data = useAppSelector(selectUnculledZones).zones;
+  var cez = <></>;
+
+  if (!data) {
+    console.log("Empty response when loading culling exclusion zones");
+  } else {
+    cez = (
+      <CullingExclusionCircles zones={data}></CullingExclusionCircles>
```

Comment:
I noticed that adding a key here can bring the culling option back to the bottom of the list like it used to be, but <ins>it can't be variable</ins>! When it's variable, unculled zones can simply disappear from the map upon receiving updates, forcing you to turn the option back on.